### ISSUE TITLE
Suggestion | Can we make enhancement in this Plugin which support Meta Audience Network Mediation Group

### ISSUE THREAD

**Issue by tusharsjagdale** [marcojak/MauiMTAdmob#99]
It will be great if this plugin support meta audience network ads. Most of the apps now using mediation group to show ads.

**Comment by marcojak** [comment]
Yes, it's something I've done in the past for a client and I could add to the plugin. 
I'll see if I can add it in one of the following releases

**Comment by tusharsjagdale** [comment]
> Yes, it's something I've done in the past for a client and I could add to the plugin. I'll see if I can add it in one of the following releases

Great. Thanks

**Comment by oliviobecker** [comment]
I would buy a license if so!

**Comment by eyeveye** [comment]
Can this plugin handle the mediation?? @marcojak

**Comment by marcojak** [comment]
Let me investigate this.

I think some of the mediations work without direct interaction with this plugin, they are directly used by Admob.
If there is something I can add here to support this, I'll have a look and add it here.

**Comment by AlessandroLanzoni1991** [comment]
@marcojak have any news on Mediations? I not understand if this plugin support mediation that not need any other SDK or mapping
